addons/website_sale_slides/static/src/xml/website_slides_unsubscribe.xml

```diff
@@ -4,10 +4,12 @@
     <t t-inherit="website_slides.SlideUnsubscribeDialog" t-inherit-mode="extension">
         <xpath expr="//t[@t-if=&quot;state.mode === 'leave'&quot;] //p[last()]" position="after">
             <t t-if="props.enroll === 'payment'">
-                <p class="alert alert-warning">
-                    <i class="fa fa-exclamation-triangle fa-3x float-start me-3"></i>
-                    This course is paid.<br/>
-                    Leaving the course and re-enrolling afterwards means that you'll be charged again.
+                <p class="d-flex align-items-center alert alert-warning">
+                    <i class="fa fa-exclamation-triangle fa-3x me-3"></i>
+                    <p class="mb-0">
+                        This course is paid.<br/>
+                        Leaving the course and re-enrolling afterwards means that you'll be charged again.
+                    </p>
```

Comment:
Thanks for the fix @Julien00859  😃 
Almost perfect for someone using Bootstrap for the first time.
Some little suggestions that can be made...

```html
<p class="d-flex align-items-center gap-3 alert alert-warning mb-0">
	<i class="fa fa-exclamation-triangle fa-3x"></i>
	<div>
		This course is paid.<br/>
		Leaving the course and re-enrolling afterwards means that you'll be charged again.
	</div>
</p>
``` 

We could go for a simple div instead of a <p> tag with `mb-0`, 
and I see that the `alert` class adds padding on the bottom so we can remove the same with the `mb-0`.
and at last, I have used `gap-3` on the outer div instead of `me-3` on the icon (in case we add anything afterward, the gap will be there).

Again Thanks and cheers!! 🍷